### PR TITLE
Fix bokeh package (add xyzservices, include pandas as runtime dependency)

### PR DESCRIPTION
**Pull request by bartbroere**
Currently, importing and using `bokeh` fails because of the missing `xyzservices` package. I realise it's a pure-Python package, but hopefully it's okay if it fixes one of the already included packages.

The quickest way to reproduce the error is:
```python
from bokeh.plotting import figure
```

I'll also try to add a regression test to detect when bokeh might break in the future for similar reasons.

### Checklists
- [x] Add a [CHANGELOG](https://github.com/pyodide/pyodide/blob/main/docs/project/changelog.md) entry
- [x] Add / update tests
- [x] Add new / update outdated documentation

### CODE REVIEW

**Comment by bartbroere** [comment]
By adding the test I found out `pandas` is a runtime dependency as well, so I included that change in the `meta.yaml` as well

+ https://app.circleci.com/pipelines/github/pyodide/pyodide/5720/workflows/9f36387b-dc94-47a9-9418-03ddcc4799ed/jobs/72066
+ https://docs.bokeh.org/en/latest/docs/first_steps/installation.html#installing-required-dependencies

File: docs/project/changelog.md [hunk 1]
```diff
@@ -22,15 +22,15 @@ myst:
   present also on `Task`s as well as `Future`s.
   {pr}`3748`
 
-- {{ Enhancement }} Added `headers` property to `pyoidide.http.FetchResponse`.
+- {{ Enhancement }} Added `headers` property to `pyodide.http.FetchResponse`.
```
Comment (ryanking13) [inline docs/project/changelog.md:25]: Nice catch, thanks!

File: packages/xyzservices/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package:
+  name: xyzservices
```
Comment (ryanking13) [inline packages/xyzservices/meta.yaml:2]: Could you add some basic tests for xyzservices?

**Review by ryanking13 — APPROVED** [review]
Thanks @bartbroere

File: packages/xyzservices/meta.yaml (hunk 2 above)
Comment (bartbroere) [reply packages/xyzservices/meta.yaml:2]: Yes, no problem. I'll add them to the PR.

File: packages/bokeh/test_bokeh.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+def test_bokeh(selenium):
```
Comment (hoodmane) [inline packages/bokeh/test_bokeh.py:1]:
```suggestion
from pytest_pyodide.decorator import run_in_pyodide

@run_in_pyodide(packages=["bokeh"])
def test_bokeh(selenium):
```
please look at what the other package tests do and imitate them.

File: packages/xyzservices/test_xyzservices.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
+def test_xyzservices(selenium):
```
Comment (hoodmane) [inline packages/xyzservices/test_xyzservices.py:1]: Also needs to use @run_in_pyodide

**Review by ryanking13 — APPROVED** [review]
Thanks, @bartbroere!

**Pull request merged**